app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -717,54 +734,124 @@ class NimbusExperimentSerializer(
+        required=False,
+    )
+    riskRevenue = serializers.BooleanField(
+        source="risk_revenue",
+        required=False,
+    )
+    statusNext = serializers.ChoiceField(
+        source="status_next",
+        choices=NimbusExperiment.Status.choices,
+        required=False,
+        allow_null=True,
+    )
+    takeawaysSummary = serializers.CharField(
+        source="takeaways_summary",
+        required=False,
+    )
+    targetingConfigSlug = serializers.CharField(
+        source="targeting_config_slug",
+        required=False,
+    )
+    totalEnrolledClients = serializers.IntegerField(
+        source="total_enrolled_clients",
+        required=False,
+    )
 
     class Meta:
         model = NimbusExperiment
         fields = [
             "application",
-            "changelog_message",
+            "changelogMessage",
             "channel",
-            "conclusion_recommendation",
+            "conclusionRecommendation",
             "countries",
-            "documentation_links",
-            "feature_config",
-            "feature_configs",
-            "warn_feature_schema",
-            "firefox_min_version",
-            "firefox_max_version",
+            "documentationLinks",
+            "featureConfig",
+            "featureConfigs",
+            "firefoxMaxVersion",
+            "firefoxMinVersion",
             "hypothesis",
-            "is_rollout",
-            "is_archived",
-            "is_enrollment_paused",
-            "is_sticky",
-            "is_first_run",
-            "locales",
+            "isArchived",
+            "isEnrollmentPaused",
+            "isFirstRun",
+            "isRollout",
+            "isSticky",
             "languages",
+            "locales",
             "name",
-            "population_percent",
-            "primary_outcomes",
-            "proposed_duration",
-            "proposed_enrollment",
-            "public_description",
-            "publish_status",
-            "reference_branch",
-            "risk_brand",
-            "risk_mitigation_link",
-            "risk_partner_related",
-            "risk_revenue",
-            "secondary_outcomes",
+            "populationPercent",
+            "primaryOutcomes",
+            "proposedDuration",
+            "proposedEnrollment",
+            "publicDescription",
+            "publishStatus",
+            "referenceBranch",
+            "riskBrand",
+            "riskMitigationLink",
+            "riskPartnerRelated",
+            "riskRevenue",
+            "secondaryOutcomes",
             "slug",
-            "status_next",
             "status",
-            "takeaways_summary",
-            "targeting_config_slug",
-            "total_enrolled_clients",
-            "treatment_branches",
+            "statusNext",
+            "takeawaysSummary",
+            "targetingConfigSlug",
+            "totalEnrolledClients",
+            "treatmentBranches",
+            "warnFeatureSchema",
```

Comment:
I really liked 🎉  the sorted fields it increases readability 👀 , I think we should always try to write/add fields in a sorted way